wrappers/ios/Sources/Wrapper/Reader/ZXIPosition+Helper.mm

```diff
@@ -0,0 +1,17 @@
+// Copyright 2022 KURZ Digital Solutions GmbH
+//
+// SPDX-License-Identifier: Apache-2.0
+
+
+#import "ZXIPosition+Helper.h"
```

Comment:
That is a bit of weird name. Why not simply put this helper in `ZXIPosition.h`?

Reply:
This is a category/extension on ZXIPosition. It's common in Objective-C and swift to make this clear by using a + within the name. 
https://stackoverflow.com/questions/12938577/why-do-some-objective-c-file-names-have-a-minus-or-plus-sign-in-their-name

I have to create an extra file as the api of the category uses one of the types defined within your c++ library. If we want a type to be available in Swift it must not use c++ specific types. 
The hellper file (all of them) are therefore only visible within the wrapper.

Reply:
Thanks for the explanation.